detox/src/devices/drivers/android/emulator/helpers/EmulatorLauncher.js

```diff
@@ -30,10 +37,13 @@ class EmulatorLauncher extends AndroidDeviceLauncher {
 
   async shutdown(adbName) {
     await this._notifyPreShutdown(adbName);
-    const port = _.split(adbName, '-')[1];
-    const telnet = this._telnetGeneratorFn();
-    await telnet.connect(port);
-    await telnet.kill();
```

Comment:
If this indeed proved to be unneeded, then we could probably remove all of `EmulatorTelnet` and `telnet-client` as a dependency (need to verify this first - obviously)

Reply:
Not likely to happen — you must have forgotten that we query AVD names via `telnet`. 🤷

Reply:
Removed it as a local dependency in the class.

Reply:
Exactly. I later had a face-palm moment regarding this, but couldn't find the conversation itself 🤦🏻